packages/cinterop/src/nativeDarwin/kotlin/io/realm/kotlin/internal/interop/RealmInterop.kt

```diff
@@ -158,9 +158,9 @@ private fun throwOnError() {
                 errorCodeNativeValue = error.error.value.toInt(),
                 messageNativeValue = error.message?.toKString(),
                 path = error.path?.toKString(),
-                userError = error.usercode_error?.asStableRef<Throwable>()?.get()
+                userError = error.user_code_error?.asStableRef<Throwable>()?.get()
             ).also {
-                error.usercode_error?.let { disposeUserData<Throwable>(it) }
+                error.user_code_error?.let { disposeUserData<Throwable>(it) }
```

Comment:
Why are we releasing this just after creating it?

Reply:
`asStableRef` does not create the pointer, only wraps a `CPointer` as a stable reference. 

The `StableRef` is created [here](https://github.com/realm/realm-kotlin/blob/5f8e6c7e74c01841caf387b9812280efef7ecd7e/packages/cinterop/src/nativeDarwin/kotlin/io/realm/kotlin/internal/interop/RealmInterop.kt#L284), `StableRef.create` actually creates the underlying pointer that is released [here](https://github.com/realm/realm-kotlin/blob/5f8e6c7e74c01841caf387b9812280efef7ecd7e/packages/cinterop/src/nativeDarwin/kotlin/io/realm/kotlin/internal/interop/RealmInterop.kt#L161).